CodeStyleExercise/ColorsViewController.swift

```diff
@@ -8,55 +8,52 @@
 
 import UIKit
 
-class ColorsViewController : UIViewController, UICollectionViewDataSource, UITextFieldDelegate
+class ColorsViewController :
+    UIViewController,
+    UICollectionViewDataSource,
+    UITextFieldDelegate
 {
-
-    class constants {
-        static let DFLTNUMCOLORS:Int=12
-    }
-
+    
     @IBOutlet weak var collectionView:UICollectionView!
     @IBOutlet weak var colorCountTextField: UITextField!
 
-    var ct: Int? = 12
+    var colorCount = 12
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Cell")
-
+        self.collectionView.register(UINib(nibName: "ColorsCollectionViewCell",
+                                            bundle: nil),
+                        forCellWithReuseIdentifier: "Cell")
         self.colorCountTextField.placeholder = "12"
     }
-
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
+    
+    private func randomColor() -> UIColor {
```

Comment:
This is fine as a function in this class, but we could also refactor it into an extension on `UIColor` for reuse.